### ISSUE TITLE
Parsec

### ISSUE THREAD

**Issue by Kodikuu** [EdoardoLaGreca/scrtp#1]
henlo i am the mr parsec and i am here to sue you /s

In all actuality, good luck m8 😄

**Comment by EdoardoLaGreca** [comment]
Thank you so much! Borgo told me about what you told him, now I'll be fully into this project hoping that one day it will be good enough to be noticed.

**Comment by Kodikuu** [comment]
> hoping that one day it will be good enough to be noticed

You mean like uh... Now? I work for Parsec 😅

**Comment by EdoardoLaGreca** [comment]
Yes I know, I mean noticed by more people so that someone actually starts using it and spread the word
